presto-main/src/main/java/io/prestosql/server/security/KerberosConfig.java

```diff
@@ -77,4 +80,23 @@ public KerberosConfig setPrincipalHostname(String principalHostname)
         this.principalHostname = principalHostname;
         return this;
     }
+
+    public enum NameType
```

Comment:
Make this a top level class `KerberosNameType`

(sharing the Airlift one doesn't make sense since this one needs different behavior)